crates/worker/src/metric_definitions.rs

```diff
@@ -61,6 +64,16 @@ pub(crate) fn describe_metrics() {
         Unit::Seconds,
         "Time spent processing a single bifrost message"
     );
+    describe_histogram!(
+        PP_COMMIT_RECORD_BATCH_DURATION,
+        Unit::Seconds,
+        "Time spent committing the effects of an applied record batch"
+    );
```

Comment:
isn't this already covered by the perf guard on rocksdb operations? `txn-write-batch` on the pp's column family.

Reply:
Yes, you are right. I will remove the unnecessary histogram.